CHANGELOG.md

```diff
@@ -1,30 +1,54 @@
-##  Next Release
+##  2021-03-02 v0.8
+
 Release highlights:
 
-- 
+- Working demonstration and integration of [`17/WAKU-RLN-RELAY`](https://rfc.vac.dev/spec/17/) in the Waku v2 `toy-chat` application
+- Beta support for ambient peer discovery using [a version of Discovery v5](https://github.com/vacp2p/rfc/pull/487)
+- A fix for the issue that caused a `store` node to run out of memory after serving a number of historical queries
+- Ability to configure a `dns4` domain name for a node and resolve other dns-based `multiaddrs`
 
 The full list of changes is below.
 
 ### Features
 
-- The `waku-rln-relay` now supports spam-protection for a specific combination of `pubsubTopic` and `contentTopic` (available under the `rln` compiler flag).  
-- The `waku-rln-relay` protocol in integrated into `chat2` (available under the`rln` compiler flag)
+- [`17/WAKU-RLN-RELAY`](https://rfc.vac.dev/spec/17/) implementation now supports spam-protection for a specific combination of `pubsubTopic` and `contentTopic` (available under the `rln` compiler flag).  
+- [`17/WAKU-RLN-RELAY`](https://rfc.vac.dev/spec/17/) integrated into chat2 `toy-chat` (available under the `rln` compiler flag)
 - Added support for resolving dns-based `multiaddrs`
+- A Waku v2 node can now be configured with a domain name and `dns4` `multiaddr`
+- Support for ambient peer discovery using [`33/WAKU-DISCV5`](https://github.com/vacp2p/rfc/pull/487)
 
 ### Changes
 
-- A new type `Timestamp` for all timestamps is introduced (currently an alias for int64).
-- All timestamps now have nanosecond resolution.
-- `waku-store` protocol identifier is updated to `/vac/waku/store/2.0.0-beta4`
+- Metrics: now monitoring content topics and the sources of new connections
+- Metrics: improved default fleet monitoring dashboard
+- Introduced a `Timestamp` type (currently an alias for int64).
```

Comment:
Should we mention the change in protobuffer timestamp field to 10?

Reply:
Ah, sorry! Saw this comment and forgot to respond. Added to [the release notes](https://github.com/status-im/nim-waku/releases/tag/v0.8) and will update in CHANGELOG in subsequent PR.